shap/benchmark/perturbation.py

```diff
@@ -93,15 +101,12 @@ def score(self, attributions, X, y=None, label=None, silent=False):
             
         self.score_values.append(np.array(svals))
         
-        if self.sort_order == "negative":
-            curve_sign = -1
-        else:
-            curve_sign = 1
-        
-        self.score_aucs.append(np.array([
-            sklearn.metrics.auc(np.linspace(0, 1, len(svals[i])), curve_sign*(svals[i] - svals[i][0]))
-            for i in range(len(svals))
-        ]))
+        if self.sort_order == "negative": curve_sign = -1
+        else: curve_sign = 1
```

Comment:
hmm this seems like it would be a flake8/pep issue, I'm surprised this isn't caught

Reply:
I'm not entirely sure if flake8/pep is called, but I will change this